### ISSUE TITLE
[Bug] env variable disappear in URL after deleting 1 character

### ISSUE THREAD

**Issue by anton-arsentyev** [a85/POSTMan-Chrome-Extension#2895]
1. Postman Version:  **4.10.5**
 2. App (Chrome app or Mac app): **Mac App**
 3. OS details: **MacOS Sierra**
 4. Is the Interceptor on and enabled in the app: **NO**
 5. Did you encounter this recently, or has this bug always been there:  **recently**
 6. Expected behaviour: **only 1 character removed**
 7. Console logs:

```
requester.js:211712  ================================================================
     o
   (/) P O S T M A N
   ^

   Go to View -> Show Postman Console to check your script logs. 

================================================================
sails.io.js:142 🌏 sync service connectivity initiated
sails.io.js:759 socket replaying queued commands
sails.io.js:142 🌏 connected to sync service.
file:///Applications/Postman.app/Contents/Resources/app/html/mode-auto.js Failed to load resource: net::ERR_FILE_NOT_FOUND
requester.js:249502 🐶 watchdog checking connection health
```

 8. Screenshots / steps:

We have non-existing env variable in URL:
![screen shot 2017-04-04 at 13 54 28](https://cloud.githubusercontent.com/assets/22887126/24655391/63f7c3fa-193e-11e7-8d1f-7cf63a6ec7a8.png)

After removing 1 symbol from variable, whole name removed:
![screen shot 2017-04-04 at 13 54 38](https://cloud.githubusercontent.com/assets/22887126/24655392/63fb6cf8-193e-11e7-8a6d-204b7131f084.png)

**Comment by thai-verys** [comment]
This is a serious bug and is slowing down development.

**Comment by a85** [comment]
We are pushing out a bug fix release soon.

**Comment by IllyaMoskvin** [comment]
Agreed with @thai-verys. Before seeing this topic, I thought this was something that occurred only when <kbd>Backspace</kbd> was pressed while the cursor was at the end of a variable. I was afraid that this was a "it's not a bug, it's a feature" sort of situation.

To be clear, if the situation I encountered is considered desirable, I'd argue strongly against it. Pressing <kbd>Backspace</kbd> at the end of the variable should delete one character, not the entire variable. I think doing otherwise would (1) go against user expectations, and (2) remove the user's power of choice.

If I _want_ to delete text one-word-at-a-time, I can press <kbd>Ctrl</kbd>+<kbd>Backspace</kbd>, or <kbd>Option</kbd>+<kbd>Delete</kbd> on Mac. Otherwise, I just press <kbd>Backspace</kbd> and hold on. In this instance, that choice would be taken from me. It's like setting `target="_blank"` on `<a/>` tags, instead of trusting the user to open the link in a new tab if they _want_ to do so.

With that in mind, I'm elated to hear that this is considered to be a bug, not a feature.

**Comment by AnaGomes** [comment]
Hi all, is there any update on this @prashantagarwal  please?

**Comment by prashantagarwal** [comment]
@AnaGomes Yes. This bug is fixed internally and on Canary builds. We are soon releasing an update next week with the fix and many new cool features. I will update this thread once we release the update.

**Comment by numaanashraf** [comment]
@AnaGomes @IllyaMoskvin @thai-verys @anton-arsentyev This is fixed in v4.11.0
